--- arts-1.5.7/artsc/artsdsp.c.glibc	2005-10-10 09:52:27.000000000 -0500
+++ arts-1.5.7/artsc/artsdsp.c	2007-08-03 07:29:48.000000000 -0500
@@ -203,7 +203,7 @@ static int is_sound_device(const char *p
   return 0;
 }
 
-int open (const char *pathname, int flags, ...)
+int (open) (const char *pathname, int flags, ...)
 {
   va_list args;
   mode_t mode = 0;
@@ -594,7 +594,7 @@ ssize_t write (int fd, const void *buf, 
   return 0;
 }
 
-ssize_t read (int fd, void *buf, size_t count)
+ssize_t (read) (int fd, void *buf, size_t count)
 {
   CHECK_INIT();
 
